samples/net/throughput_server/README.rst

```diff
@@ -0,0 +1,60 @@
+.. _throughput-server-sample:
+
+Throughput Server
+#################
+
+Overview
+********
+
+The throughput-server sample application for Zephyr implements an UDP server
```

Comment:
I don't know if @dbkinder would notice, but "an UDP" should be "a UDP", an article form is used based on the pronunciation, not on the written form, and before the [j] sound, English requires "a", not "an", e.g. "a young man".